recipes/libtiff/all/conanfile.py

```diff
@@ -37,6 +40,7 @@ class LibtiffConan(ConanFile):
         "lzma": True,
         "jpeg": "libjpeg",
         "zlib": True,
+        "lerc": True,
```

Comment:
honestly it could be disabled by default

Reply:
Yeah, I it's a quite uncommon compression format. Would allow the PR to pass without waiting for the LERC fix to get merged either.